app/javascript/mastodon/api.ts

```diff
@@ -66,3 +83,17 @@ export default function api(getState: GetState) {
     ],
   });
 }
+
+export async function apiRequest<ApiResponse = unknown>(
+  method: Method,
+  url: string,
+  params?: unknown,
```

Comment:
Would params be a `Record<string, unknown>`? (I'm assuming these are query parameters?)

Reply:
`Axios` uses `any` for this type, so I preferred to not force it to anything (and use `unknown` rather than `any` as a best practice)

Reply:
Hmm, okay!